FRAuth/FRAuth/Authentication/Node.swift

```diff
@@ -42,26 +42,27 @@ public class Node: NSObject {
     var serverConfig: ServerConfig
     /// OAuth2Client information for AuthService/Node API communication
     var oAuth2Config: OAuth2Client?
-    /// Optional SessionManager for SDK's abstraction layer
-    var sessionManager: SessionManager?
     /// TokenManager instance to manage, and persist authenticated session
     var tokenManager: TokenManager?
+    /// KeychainManager instance to persist, and retrieve credentials from storage
+    var keychainManager: KeychainManager?
     
     
     
     //  MARK: - Init
     
     /// Designated initialization method for Node; AuthService process will initialize Node with given response from OpenAM
-    ///
     /// - Parameters:
     ///   - authServiceId: Unique UUID for current AuthService flow
     ///   - authServiceResponse: JSON response object of AuthService in OpenAM
     ///   - serverConfig: ServerConfig object for AuthService/Node communication
     ///   - serviceName: Service name for AuthService (TreeName)
     ///   - authIndexType: authIndexType value in AM (default to 'service')
     ///   - oAuth2Config: (Optional) OAuth2Client object for AuthService/Node communication for abstraction layer
-    /// - Throws: AuthError error may be thrown from parsing AuthService response, and parsing Callback(s)
-    init?(_ authServiceId: String, _ authServiceResponse: [String: Any], _ serverConfig: ServerConfig, _ serviceName: String, _ authIndexType: String, _ oAuth2Config: OAuth2Client? = nil, _ sessionManager: SessionManager? = nil, _ tokenManager: TokenManager? = nil) throws {
+    ///   - keychainManager: KeychainManager instance to manage and persist authenticated session
+    ///   - tokenManager: TokenManager  instance to manage and persist authenticated session
```

Comment:
These comments make it seem like the `keychainManager` and `tokenManager` do the exact same thing. Is that intentional?

Reply:
Thanks for pointing that out! I missed that part, and wasn't intentional. Just updated the description.